lib/js/components/TextGroup/TextGroup.vue

```diff
@@ -0,0 +1,296 @@
+<template>
+	<div
+		class="ds-textGroup"
+		:class="{
+			'-ds-small': size === TEXT_GROUP_SIZES.SMALL,
+
+			'-ds-hovered': state === TEXT_GROUP_STATES.HOVERED,
+			'-ds-loading': isLoading,
+			'-ds-disabled': state === TEXT_GROUP_STATES.DISABLED,
+
+			'-ds-interactive': isInteractive,
+			'-ds-selected': isSelected,
+
+			[loadingSizeClassName]: isLoading,
+
+			[colorClassName]: true,
+		}"
+	>
+		<div
```

Comment:
when `eyebrowtext:null` height of `loading` and `default` state of entire component doesn't match

Reply:
the same problem for other combinations of null text props

Reply:
https://github.com/bethinkpl/design-system/pull/401/commits/868b28ddd6ec59f2005867c8a763683b8c7ead24
+ 
https://bethinkteam.slack.com/archives/CFBLF5RGA/p1723109726737899?thread_ts=1723020458.153299&cid=CFBLF5RGA